### PR TITLE
Fix parameter keys and add the new parameter 'router.connection.idle.…

### PR DESCRIPTION
**Pull request by anwar6953**
Fix parameter keys and add the new parameter 'router.connection.idle.timeout.secs' to cdap-site.rst

### CODE REVIEW

File: cdap-docs/admin-manual/source/installation/cdap-site.rst [hunk 1]
```diff
@@ -130,40 +130,43 @@ see the :ref:`configuration-security` section.
...
      - ``3600``
      - Timeout value in seconds for a SQL operation which has no more results to be fetched
-   * - ``gateway.boss.threads``
+   * - ``router.boss.threads``
      - ``1``
      - Number of Netty server boss threads
-   * - ``gateway.connection.backlog``
+   * - ``router.connection.idle.timeout.secs``
+     - ``15``
+     - The number of seconds after an http request completes that idle router connections are closed
```
Comment (awholegunch) [inline cdap-docs/admin-manual/source/installation/cdap-site.rst]: HTTP

**Comment by awholegunch** [comment]
One comment, otherwise LGTM.

File: cdap-docs/admin-manual/source/installation/cdap-site.rst (hunk 1 above)
Comment (anwar6953) [inline cdap-docs/admin-manual/source/installation/cdap-site.rst]: updated the rst file.
Note that its lowercased in on develop and release/3.3 branches, though. Probably not a big deal.

**Comment by awholegunch** [comment]
LGTM

**Comment by anwar6953** [comment]
Quick build running off this branch: http://builds.cask.co/browse/CDAP-DQBP38-1

**Pull request merged**